data/conversion_factors_heating.csv

```diff
@@ -3,5 +3,5 @@
 1,liquid_gas,kg,14.1
 2,coal,kg,6.0
 3,pellet,kg,5.4
-4,woodchips,kg,5.2
+4,wood chips,kg,5.2

```

Comment:
I would keep it as one word - or alternatively "wood_chips" (similar to "liquid_gas"